src/server_manager/electron_app/write_environment_variables.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -eu
```

Comment:
Let's rename this to something like `write_production_environment.sh` to make it clear that it's only for production.

Reply:
Renamed to `write_release_environment_action.sh` for consistency with the other scripts.

Reply:
Maybe `write_production_environment_action.sh` then? This is not the environment for the release (your machine/travis). It's the environment in production (the manager's machine)

Up to you.